<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationDetailsWidget</class>
 <widget class="QWidget" name="StationDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Станция</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Индекс:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="indexLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Код ИКАО:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ccccLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Название (русское):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="nameRuLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Название (короткое):</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="nameShortLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Название (международное):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="nameEnLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Страна:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="countryComboBox">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Тип:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="stationTypeComboBox">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Направление ВПП:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="airstripDirectionLineEdit"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Уровень поймы:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="poimaLevelLineEdit"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Уровень затопления:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="floodLevelLineEdit"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Опасный уровень:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="dangerLevelLineEdit"/>
     </item>
     <item row="13" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>508</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveChangesPushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Сохранить изменения&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <widget class="GeoPointEditor" name="geoPointEditor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="oneLine">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Координаты:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GeoPointEditor</class>
   <extends>QWidget</extends>
   <header location="global">meteo/commons/ui/custom/geopointeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
